astronomer/providers/google/cloud/extractors/bigquery_insertjoboperatorasync_extractor.py

```diff
@@ -0,0 +1,78 @@
+from typing import Any, List, Optional
+
+import sqlalchemy
+from airflow.models.taskinstance import TaskInstance
+from airflow.utils.log.logging_mixin import LoggingMixin
+from google.cloud.bigquery import Client
+from openlineage.airflow.extractors.base import BaseExtractor, TaskMetadata
+from openlineage.airflow.utils import get_job_name
+from openlineage.common.provider.bigquery import BigQueryDatasetsProvider
+
+from astronomer.providers.google.cloud.operators.bigquery import (
+    BigQueryInsertJobOperatorAsync,
+)
+
+
+class BigQueryInsertJobOperatorAsyncExtractor(BaseExtractor, LoggingMixin):
```

Comment:
try to shorten the class name and file name please

Reply:
Done